### ISSUE TITLE
LWJGL out of date

### ISSUE THREAD

**Issue by YouYouTheBoxx** [Blaze4D-MC/Blaze4D#48]
Can't load minecraft 1.17 because of:

```
java.lang.NoSuchMethodError: 'org.lwjgl.system.SharedLibrary org.lwjgl.system.Library.loadNative(java.lang.Class, java.lang.String, org.lwjgl.system.Configuration, java.lang.String[])'
	at org.lwjgl.vulkan.VK.create(VK.java:54)
	at org.lwjgl.vulkan.VK.<clinit>(VK.java:36)
	at org.lwjgl.vulkan.VK10.nvkEnumerateInstanceLayerProperties(VK10.java:3999)
	at org.lwjgl.vulkan.VK10.vkEnumerateInstanceLayerProperties(VK10.java:4056)
	at me.hydos.rosella.Rosella.getSupportedValidationLayers(Rosella.java:121)
	at me.hydos.rosella.Rosella.validationLayersSupported(Rosella.java:110)
	at me.hydos.rosella.Rosella.<init>(Rosella.java:54)
	at me.hydos.rosella.Rosella.<init>(Rosella.java:49)
	at net.minecraft.class_1041.handler$zpf000$initializeRosellaWindow(class_1041.java:636)
	at net.minecraft.class_1041.<init>(class_1041.java:119)
	at net.minecraft.class_3682.method_16038(class_3682.java:21)
	at net.minecraft.class_310.<init>(class_310.java:481)
	at net.minecraft.client.main.Main.main(Main.java:179)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:234)
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:153)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28)
```

Compiled with latest 'master' branch.

**Comment by OroArmor** [comment]
What operating system?

**Comment by OroArmor** [comment]
Oh wait. It's lwjgl reee minecraft bad

**Comment by YouYouTheBoxx** [comment]
> What operating system?

Windows 10 with Java [1.8.0_291 / OpenJDK 15 / OpenJDK 16]

**Comment by hYdos** [comment]
possible solution is repackaging lwjgl and using the newer version for vulkan

**Comment by YouYouTheBoxx** [comment]
> possible solution is repackaging lwjgl and using the newer version for vulkan

and how i can do that ?

**Comment by hYdos** [comment]
you can't. you would need to wait for one of us to do it

**Comment by YouYouTheBoxx** [comment]
> you can't. you would need to wait for one of us to do it

uhmm i see ok. thanks anyway.

**Comment by DioEgizio** [comment]
getting same error

**Comment by SirObby** [comment]
Its LWJGL 3.2.2 bad 3.3.0 good.
I might try to change some configs for MultiMC to use LWJGL 3.3.0. I might share it once its done with a yutorial

**Comment by DioEgizio** [comment]
> 
> 
> Its LWJGL 3.2.2 bad 3.3.0 good.
> I might try to change some configs for MultiMC to use LWJGL 3.3.0. I might share it once its done with a tutorial.

thanks, if you are able could send how to do? Because I tried but failed

**Comment by SirObby** [comment]
If I can change the LWJGL version in MultiMC Ill do a tutorial.

**Comment by SirObby** [comment]
> > Its LWJGL 3.2.2 bad 3.3.0 good.
> > I might try to change some configs for MultiMC to use LWJGL 3.3.0. I might share it once its done with a tutorial.
> 
> thanks, if you are able could send how to do? Because I tried but failed

I decided not to do it.

**Comment by ramidzkh** [comment]
Possibly fixed by 9e450beeb49f4030c74cd3feebb867ba37f235e4, please try. I'll close this issue once confirmed fixed for other people.
